modules/cert-manager-override-arguments.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-customizing-api-fields.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-override-arguments_{context}"]
+= Customizing cert-manager by overriding arguments from the cert-manager Operator API
+
+You can override the supported arguments for the {cert-manager-operator} by adding a `spec.controllerConfig` section in the `CertManager` resource.
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the `cluster-admin` role.
+* The {cert-manager-operator} 1.11.1 or later is installed.
+
+.Procedure
+
+. Edit the `CertManager` resource by running the following command:
++
+[source,terminal]
+----
+$ oc edit certmanager cluster
+----
+
+. Add a `spec.controllerConfig` section with the following override arguments:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: CertManager
+metadata:
+  name: cluster
+  ...
+spec:
+  ...
+  controllerConfig: <1>
+    overrideArgs:
+      - '--dns01-recursive-nameservers=<host>:<port>' <2>
+      - '--dns01-recursive-nameservers-only' <3>
+      - '--acme-http01-solver-nameservers=<host>:<port>' <4>
+      - '--v=<verbosity_level>' <5>
+      - '--metrics-listen-address=<host>:<port>' <6>
+  webhookConfig:
+    overrideArgs:
+      - '--v=4'
+  cainjectorConfig:
+    overrideArgs:
+      - '--v=2'
+----
+<1> Add the `spec.controllerConfig` section.
+<2> Provide a comma-separated list of `<host>:<port>` nameservers to query for the DNS-01 self check. For example, `--dns01-recursive-nameservers=1.1.1.1:53`.
+<3> Specify to only use recursive nameservers instead of checking the authoritative nameservers associated with that domain.
+<4> Provide a comma-separated list of `<host>:<port>` nameservers to query for the ACME HTTP01 self check. For example, `--acme-http01-solver-nameservers=1.1.1.1:53`.
+<5> Specify to set the log level verbosity to determine the verbosity of log messages.
+<6> Specify the host and port for the metrics endpoint. The default value is `--metrics-listen-address=0.0.0.0:9402`.
+
+. Save your changes and quit the text editor to apply your changes.
+
+.Verification
+
+* Verify that arguments are updated for cert-manager pods by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n cert-manager -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+...
+  metadata:
+    name: cert-manager-6d4b5d4c97-kldwl
+    namespace: cert-manager
+...
+  spec:
+    containers:
+    - args:
+      - --acme-http01-solver-nameservers=1.1.1.1:53
+      - --cluster-resource-namespace=$(POD_NAMESPACE)
+      - --dns01-recursive-nameservers=1.1.1.1:53
+      - --dns01-recursive-nameservers-only
+      - --leader-election-namespace=kube-system
+      - --max-concurrent-challenges=60
+      - --metrics-listen-address=<host>:<port>
```

Comment:
since this is example output, I think it would be better to have an actual host and port values. Up to you though.